server/models/userModel.js

```diff
@@ -27,6 +27,23 @@ const userSchema = new Schema(
       type: String,
       required: false,
     },
+    location: {
+      type: {
+        type: String,
+        enum: ["Point"],
+      },
+      coordinates: {
+        type: [Number],
+        index: "2dsphere",
+      },
+    },
+    bio: String,
+    profilePictureUrl: {
+      type: String,
+    },
+    cuisines: {
+      type: [String],
+    },
   },
```

Comment:
I think we should add a notifications field here. This will keep persistent track of all that user's notifications. It can look something like this:

`
    {
      type: "message", //either "message" or "order"
      name: "Mike Leech", //Name of who the notification is from
      preview: "Sounds good. I'll see you soon.", //The latest message
      link: "/messages/57c341465g15dgf54510552", //The link the user should click to be taken to the relevant page for that notification
      read: false //Whether the user has already seen the notification or not, to be used for UI purposes
    },
`

Let me know what you think